starknet_py/net/signer/test_stark_curve_signer.py

```diff
@@ -8,8 +8,9 @@
 from starknet_py.net.models.transaction import Declare, DeployAccount, Invoke
 from starknet_py.net.signer.stark_curve_signer import KeyPair, StarkCurveSigner
 
-contract_source = (
-    Path(os.path.dirname(__file__)) / "../../tests/e2e/mock/contracts/erc20.cairo"
+compiled_contract = (
+    Path(os.path.dirname(__file__))
+    / "../../tests/e2e/mock/contracts_compiled/erc20_compiled.json"
```

Comment:
Can we do something about this path? Could it be somehow changed to relative to project root?

Reply:
We could change it to a fixture, but that would require indirect parametrisation of a fixture below. I can open an issue for that.

Reply:
Let's do something about it.